### PR TITLE
docs: change examples within services to be tree-shakeable

### PR DESCRIPTION
**Pull request by matsko**

### CODE REVIEW

File: aio/content/examples/toh-pt6/src/app/app.module.ts [hunk 1]
```diff
@@ -49,7 +47,7 @@ import { MessagesComponent }    from './messages/messages.component';
     HeroSearchComponent
     // #docregion v1
   ],
-  providers: [ HeroService, MessageService ],
+  providers: [ ],
```
Comment (IgorMinar) [inline aio/content/examples/toh-pt6/src/app/app.module.ts]: why not remove?

File: aio/content/examples/toh-pt5/src/app/app.module.ts [hunk 2]
```diff
@@ -30,7 +28,7 @@ import { AppRoutingModule }     from './app-routing.module';
     MessagesComponent
   ],
   // #enddocregion dashboard
-  providers: [ HeroService, MessageService ],
+  providers: [ ],
```
Comment (IgorMinar) [inline aio/content/examples/toh-pt5/src/app/app.module.ts]: remove?

File: aio/content/examples/universal/tsconfig.json [hunk 3]
```diff
@@ -8,6 +8,7 @@
     "emitDecoratorMetadata": true,
     "experimentalDecorators": true,
     "target": "es5",
+    "skipLibCheck": true,
```
Comment (IgorMinar) [inline aio/content/examples/universal/tsconfig.json]: why this change? seems unrelated.

**Review by IgorMinar — APPROVED** [review]
small nits, but otherwise lgtm

File: aio/content/examples/toh-pt4/src/app/app.module.ts [hunk 4]
```diff
@@ -23,9 +21,8 @@ import { MessagesComponent } from './messages/messages.component';
   // #docregion providers
   // #docregion providers-heroservice
   providers: [
-    HeroService,
     // #enddocregion providers-heroservice
-    MessageService
+    // no need to place any providers due to the `providedIn` flag...
```
[the remark this answers is not in the document]
Comment (jbogarthyde) [reply aio/content/examples/toh-pt4/src/app/app.module.ts]: The surrounding text is changed in PR #22934
